android/filament-android/src/main/java/com/google/android/filament/RenderTarget.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.filament;
+
+import android.support.annotation.IntRange;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.nio.Buffer;
+import java.nio.BufferOverflowException;
+
+public class RenderTarget {
+    private long mNativeObject;
+    private final Texture[] mTextures = new Texture[2];
+
+    private RenderTarget(long nativeRenderTarget, Builder builder) {
+        mNativeObject = nativeRenderTarget;
+        mTextures[0] = builder.mTextures[0];
+        mTextures[1] = builder.mTextures[1];
+    }
+
+    long getNativeObject() {
+        if (mNativeObject == 0) {
+            throw new IllegalStateException("Calling method on destroyed RenderTarget");
```

Comment:
Looks like you forgot to add Engine.destroyRenderTarget

Reply:
Excellent catch, fixed.